standalone-metastore/metastore-server/src/main/resources/package.jdo

```diff
@@ -345,20 +345,20 @@
       <datastore-identity>
         <column name="CD_ID"/>
       </datastore-identity>
-      <field name="cols" table="COLUMNS_V2" >
+      <field name="cols" persistence-modifier="persistent" table="COLUMNS_V2">
         <collection element-type="MFieldSchema"/>
-        <foreign-key name="COLUMNS_V2_FK" delete-action="cascade"/>
```

Comment:
We do want cascade-delete here. Any reason to remove it?

Reply:
```
If a foreign-key is specified (in MetaData) for the relation field then leave any deletion to the datastore to perform
``` 
I don't see any such relationship cascading relationship defined in the schema for Derby or MySQL, so DN should be doing it.